ambry-quota/src/main/java/com/github/ambry/quota/AmbryQuotaManager.java

```diff
@@ -70,6 +72,10 @@ public AmbryQuotaManager(QuotaConfig quotaConfig, ThrottlePolicy throttlePolicy,
     this.throttlePolicy = throttlePolicy;
     this.quotaConfig = quotaConfig;
     this.quotaMetrics = new QuotaMetrics(metricRegistry);
+    Consumer<Collection<Account>> accountUpdateQuotaConsumer = (accounts) -> {
+      onAccountUpdateNotification(accounts);
+    };
+    accountService.addAccountUpdateConsumer(accountUpdateQuotaConsumer);
```

Comment:
you can just do `accountService.addAccountUpdateConsumer(this::onAccountUpdateNotification);`.